src/docs/templates/sweepstakes.hbs

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	{{> html-head title="Sweepstakes" }}
+</head>
+
+<body>
+    {{> sections/header}}
+    <section class="sweepstakes bg-primary-light">
+        <div class="container">
+            <figure class="row m-0 align-items-center">
+                <picture class="col-lg-7 order-lg-2">
+                    <source srcset="https://via.placeholder.com/457x406/F5DADF.jpg" media="(min-width: 992px)" />
+                    <img src="https://via.placeholder.com/457x406.jpg/F5DADF" class="w-100">
+                </picture>
+                <figcaption class="px-0 col-lg-5 order-lg-1 my-lg-4">
+                    <form action="" class="px-4 py-3 bg-white text-center sweepstakes__form">
```

Comment:
this can be 1 unique DOM element maybe and we don't need need a new class only for the border-radius, there are Bootstrap utilities for that https://getbootstrap.com/docs/4.0/utilities/borders/ if default border-radius is different than the one in the design we must ask to designers which one to use because we must use only 1 border-radius value (same like the one of the buttons).